arangod/MMFiles/MMFilesCollection.cpp

```diff
@@ -1851,9 +1876,9 @@ int MMFilesCollection::iterateMarkersOnLoad(transaction::Methods* trx) {
   LOG_TOPIC(TRACE, arangodb::Logger::FIXME)
       << "found " << openState._documents << " document markers, "
       << openState._deletions << " deletion markers for collection '"
-      << _logicalCollection->name() << "'";
+      << _logicalCollection.name() << "'";
 
-  if (_logicalCollection->version() <= LogicalCollection::VERSION_30 &&
```

Comment:
yoda style ?

Reply:
it is usually recommended to keep constants on the left side of the comparison catch '==' v.s. '=' typos by the compiler